SpringProject/src/main/java/com/gradproject2019/userConference/service/UserConferenceServiceImpl.java

```diff
@@ -38,11 +45,32 @@ public UserConferenceResponseDto saveInterest(UUID token, UserConferenceRequestD
         }
     }
 
+    @Override
+    public List<ConferenceResponseDto> getConferenceByUserId(UUID token, Long userId) {
+        List<ConferenceResponseDto> conferenceResponseDtos = new ArrayList<>();
+
+        checkUserAuthorised(token);
```

Comment:
A minor point but should the token authorisation be happening in this method? I would imagine it would be better to check this in the API before this method is even called

Reply:
Yes, this point has been raised a couple of times. There is a ticket in the backlog for changing the structure of this for all of our authorisation. But this has not been pulled into this weeks sprint as we are focusing on getting as many features as possible done before making these improvements.